krun_ext_common.py

```diff
@@ -2,26 +2,233 @@
 
 import sys
 import json
-import subprocess
+from subprocess import Popen, PIPE
+from dataclasses import dataclass
+from decimal import Decimal
 import tempfile
 import csv
 import os
 
 
+PATHS = {}
+with open("paths.sh") as f:
+    for line in f:
+        k, v = line.strip().split("=")
+        PATHS[k] = v
+
+# Make sure these limits match those in the Krun config file.
```

Comment:
Could we extract them from the Krun config file perhaps?

Reply:
I'd intended on sharing lots of stuff, but the other way around: having the Krun config import this. However, I'm a little hesitant, as this is Python3 code, and Krun is Python2.

I bet it wouldn't be much effort to port Krun to Python3, but in the short term I think we can just duplicate.